upb/msg.c

```diff
@@ -252,45 +252,49 @@ static void _upb_mapsorter_getkeys(const void *_a, const void *_b, void *a_key,
   _upb_map_fromkey(b_tabkey, b_key, size);
 }
 
+#define COMPARE_INTEGERS(a, b) ((a) < (b) ? -1 : ((a) == (b) ? 0 : 1))
```

Comment:
UPB_COMPARE_INTEGERS to avoid collisions

Reply:
Done.